js/main.js

```diff
@@ -0,0 +1,32 @@
+var xhr = new XMLHttpRequest();
+var $h2 = document.querySelector('h2');
+var interval = null;
+var count = 1;
+var $cocktails = document.querySelector('.cocktails-text');
+
+xhr.open('GET', 'https://www.thecocktaildb.com/api/json/v2/9973533/popular.php');
+xhr.responseType = 'json';
+xhr.addEventListener('load', function () {
+
+  var data = xhr.response;
+  interval = setInterval(function () {
+    if (count === 11) {
```

Comment:
Rather than an arbitrary number here, can you use the length of the array of data? That way if you ever have more results, your code can adapt to different lengths.